File: store/src/main/java/fooddeliverytest/domain/Deliverystared.java
package fooddeliverytest.domain;

import fooddeliverytest.domain.*;
import fooddeliverytest.infra.AbstractEvent;
import java.util.*;
import lombok.*;

@Data
@ToString
public class Deliverystared extends AbstractEvent {

    private Long storeid;
    private String orderid;
    private String status;
    private String customerid;
    private String address;

    public Deliverystared(Ordermanagement aggregate){
        super(aggregate);
    }
    public Deliverystared(){
        super();
    }
}

File: app/Deliverystared.py

from AbstractEvent import AbstractEvent
import json

class Deliverystared(AbstractEvent):
    storeid : int
    orderid : str
    status : str
    customerid : str
    address : str
    
    def __init__(self):
        super().__init__()
        self.storeid = None
        self.orderid = None
        self.status = None
        self.customerid = None
        self.address = None

